OWML.Common/IModCombination.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+
+namespace OWML.Common
+{
+	public interface IModCombination
+	{
+		string GetCombo();
+		void _SetPressed(bool state = true);
+		float GetLastPressedMoment();
+		float GetPressDuration();
+		bool IsFirst(bool keep = false);
+		void _AddSingle(KeyCode button);
+		List<KeyCode> _GetSingles();
+		void _ClearSingles();
```

Comment:
Why are some methods starting with _?

Reply:
because they are not supposed to be used by modders

Reply:
I could move Singles stuff to InputHandler by adding Dictionary there
Can't do the same with InputHandler's service methods, they're being used by patched methods, I think I can add a warning thing though

Reply:
removed from interface and changed to internal

Reply:
Looks like you didn't push the changes yet?

Reply:
now it is